web/src/OverviewActionBar.tsx

```diff
@@ -224,6 +245,43 @@ export let ButtonRightPill = styled(ButtonRoot)`
   border-radius: 0 4px 4px 0;
 `
 
+const FilterTermTextField = styled(TextField)`
```

Comment:
When I start entering text, the width of the box changes a fair amount:
![image](https://user-images.githubusercontent.com/7453991/121557456-c2079200-c9e2-11eb-9ec0-7e1dc69e6a6f.png)
![image](https://user-images.githubusercontent.com/7453991/121557490-c764dc80-c9e2-11eb-91ac-57bc4a311046.png)
This would be nice to fix but is probably safe to ignore.

Reply:
Yeaaaah, I noticed this too. The resource filter field behaves the same, so I think it's okay to leave as-is for now.